public/components/common/Loader.tsx

```diff
@@ -3,12 +3,22 @@ import "./Loader.scss"
 import React, { useState } from "react"
 import { useTimeout } from "@fider/hooks"
 
-export function Loader() {
+interface LoaderProps {
+  text?: string
+  className?: string
+}
+
+export function Loader(props: LoaderProps) {
   const [show, setShow] = useState(false)
 
   useTimeout(() => {
     setShow(true)
   }, 500)
 
-  return show ? <div className="c-loader" /> : null
+  return show ? (
+    <div className={props.className}>
+      <div className="c-loader" />
```

Comment:
the `.c-{name} is often the top level element returned by the component. Can we add the className to the existing  div?

Reply:
Hmm I need to double check this, there may be a reason why I did it like that, or maybe adapt the Loader's SCSS to target the inner div (maybe a `c-loader__spinner` and a `c-loader__text`?)